eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionWriter.java

```diff
@@ -47,8 +48,14 @@ public AccessionWriter(SubmittedVariantAccessioningService service, AccessionRep
     @Override
     public void write(List<? extends ISubmittedVariant> variants) throws Exception {
         List<AccessionWrapper<ISubmittedVariant, String, Long>> accessions = service.getOrCreate(variants);
-        accessions.sort(new AccessionWrapperComparator(variants));
-        accessionReportWriter.write(accessions);
+
+        List<AccessionWrapper<ISubmittedVariant, String, Long>> denormalizedAccessions = new ArrayList<>();
+        accessions.forEach(accession -> denormalizedAccessions
+                .add(new AccessionWrapper(accession.getAccession(), accession.getHash(),
+                                          accessionReportWriter.denormalizeVariant(accession.getData()))));
+
+        denormalizedAccessions.sort(new AccessionWrapperComparator(variants));
+        accessionReportWriter.write(denormalizedAccessions);
```

Comment:
I know it was already this way, but given that the sorting is specific to the report, maybe it should be called first thing inside the `write` method?